nextflow.config

```diff
@@ -136,20 +136,20 @@ params {
     schema_ignore_params       = 'genomes'
 
     // Shovill options
-    assembler                  = 'spades'
+    sr_assembler                  = 'spades' // WGS_ASSEMBLY: choice of genome assembler for short read data [skesa,spades,megahit,velvet]
 
     // CheckM options
-    checkm_db                  = false
+    checkm_db                  = false // ASSEMBLY_QC: path to local CheckM lineages directory
 
     // QUAST options
-    reference                  = false
-    reference_gff              = false
-    combine_quast              = false
+    reference                  = false // ASSEMBLY_QC: path to reference genome file 
```

Comment:
@jimmyliu1326 do you think `reference` & `reference_gff` needs updating? We can specify `refernce_genome_fasta` or something like that